src/server.c

```diff
@@ -3888,7 +3888,13 @@ static void readOOMScoreAdj(void) {
  * A process_class value of -1 implies OOM_CONFIG_MASTER or OOM_CONFIG_REPLICA,
  * depending on current role.
  */
+static int oom_adj_managed_by_redis = 0;
```

Comment:
i'd rather move this static var into the function so it's not placed between the function and it's doc comment.
also maybe add some comment explaining it.
i.e. remembers if the value in `/proc` was modified by us (or restored to default)
maybe name it "adjusted_by_redis" instead of "managed_by_redis"?